woocommerce/woocommerce-android
# Replace deprecated menu calls login

Closes: #7358
Closes: #7357


### Description

* Replaces deprecated "menu" methods usage in `LoginNoWPcomAccountFoundFragment` 
* Removes unused `LoginJetpackRequiredFragment`

### Testing instructions

* Start login flow via wp.com account
* Enter random email
* Notice that menu in `LoginNoWPcomAccountFoundFragment` has not changed


### Images/gif

### before/after

<img src="https://user-images.githubusercontent.com/4923871/188655127-81494d91-a8cf-4c4a-b3e6-2e963f89a799.jpg" width=300/>
<img src="https://user-images.githubusercontent.com/4923871/188655135-66b3b59b-be74-4a9b-b140-b3a4915384d3.jpg" width=300/>
